### DISCUSSION TITLE
Bug in https://demo.ldproxy.net/vineyards/collections/vineyards/items?

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#1032] [General]
The items are not showing in the map. Is it possible to fix it soon? 
I have a presentation on a ICA Workshop and like to run a demo on this dataset.

For example: https://demo.ldproxy.net/vineyards/collections/vineyards/items?f=html&filter-lang=cql2-text&filter=region%3D%27Rheinhessen%27&limit=2

Regards
Sverre

**Comment by cportele** [comment]
Thanks for the alert. Please try again. We upgraded MapLibre and added a terrain to the map style, but this obviously causes issues for the feature pages. I have removed the terrain for now.

**Reply by Sverre-Iversen** [reply]
Thanks !
